tests/apps/search/test_indexers_courses.py

```diff
@@ -585,7 +585,7 @@ def test_indexers_courses_get_es_document_no_image_icon_picture(self):
             course_icons_placeholder,
             CategoryPlugin,
             "en",
-            **{"page": category.extended_object}
+            **{"page": category.extended_object},
         )
         course.extended_object.publish("en")
         # Make sure we associate an image-less picture with the category through
```

Comment:
You should revert this change.

Reply:
The command `make lint-back` adds this correction of a missing comma.